api/specs/web-server/_projects_crud.py

```diff
@@ -97,10 +97,11 @@ async def replace_project(project_id: ProjectID, _replace: ProjectReplace):
 
 @router.patch(
     "/projects/{project_id}",
-    response_model=Envelope[ProjectGet],
+    response_model=None,
+    status_code=status.HTTP_204_NO_CONTENT,
 )
-async def update_project(project_id: ProjectID, update: ProjectUpdate):
-    """Partial update of a project resource"""
+def patch_project(project_id: ProjectID, _new: ProjectPatch):
```

Comment:
Question: Why is this entry the only one not asynchronous?

Reply:
Typo, thanks.